_layouts/default.html

```diff
@@ -80,6 +80,12 @@
       <a href="https://www.linkedin.com/company/sel4"><img style="height:36px; padding-left:5px; padding-top:2px; padding-bottom:2px;" src={{ "/images/icons/linkedin.svg" | relative_url }} alt="linkedin icon"></a>
       <br>
       <br>
+{% if site.on_seL4 %}
+  <a href="/Info/Website/">About this web server</a>
+  Served by <a href="https://micropython.org/">MicroPython</a>
+on <a href="https://lionsos.org/"> LionsOS</a> on seL4.
+<br/>
+{% endif %}
```

Comment:
Would it makes sense to make this a small `_includes` file? I worry about duplicating the text and links here and people then later only updating one of them. (To be fair, the rest of the footer also has a bunch of duplication, so I'm not entirely sure it's worth it).

Reply:
That looks great, thanks!